ui/hooks/useOriginMetadata.js

```diff
@@ -1,14 +1,12 @@
 import { useSelector } from 'react-redux';
-import { getDomainMetadata } from '../selectors';
+import { getSubjectMetadata } from '../selectors';
 
 /**
  * @typedef {Object} OriginMetadata
  * @property {string} host - The host of the origin
```

Comment:
```suggestion
```

Reply:
Actually this is still being set below, and is derived from the origin. But I don't think we need it. It's never used.